Fits/T1 Logistics Cruiser/[NVY-30] Augoror.md

```diff
@@ -0,0 +1,43 @@
+# Augoror
+
+T1 Cruiser Logi
+
+## Description
+
+An entry option for learning capchain T1 logistics, emphasis on tank and output. Scram refit for scram-chaining to counter hostile booshers. At least 2 required to function.
+
+## Fit
+
+```
+[Augoror, [NVY-30] Augoror]
```

Comment:
More new player friendly and different tank profile
```
[Augoror, [NVY-30] Augoror]

Damage Control II
800mm Crystalline Carbonide Restrained Plates
Multispectrum Energized Membrane II
Multispectrum Energized Membrane II
Reactive Armor Hardener

10MN Monopropellant Enduring Afterburner
Alumel-Wired Enduring Sensor Booster
Alumel-Wired Enduring Sensor Booster

Medium I-ax Enduring Remote Armor Repairer
Medium I-ax Enduring Remote Armor Repairer
Medium I-ax Enduring Remote Armor Repairer
Medium Inductive Compact Remote Capacitor Transmitter
Medium Inductive Compact Remote Capacitor Transmitter

Medium Trimark Armor Pump I
Medium Trimark Armor Pump I
Medium Trimark Armor Pump I


Acolyte I x4


Nanite Repair Paste x10
ECCM Script x2
Scan Resolution Script x2
Targeting Range Script x2
Faint Epsilon Scoped Warp Scrambler x1
```

Reply:
Definitely good with this change, I was thinking the other one was a bit greedy but especially for new players it's too rough on cap. Will edit.